vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -908,6 +912,36 @@ private void createAnalysisRecords(
             });
   }
 
+  private void deleteWorkflowRun(HttpServerExchange exchange) {
+    final var vidarrId =
+        exchange.getAttachment(PathTemplateMatch.ATTACHMENT_KEY).getParameters().get("hash");
+    exchange.setStatusCode(
+        processor.delete(
+            vidarrId,
+            new DeleteResultHandler<>() {
+              @Override
+              public Integer deleted() {
+                return StatusCodes.OK;
+              }
+
+              @Override
+              public Integer internalError(SQLException e) {
+                return StatusCodes.INTERNAL_SERVER_ERROR;
+              }
+
+              @Override
+              public Integer noWorkflowRun() {
+                return StatusCodes.NOT_FOUND;
+              }
+
+              @Override
+              public Integer stillActive() {
+                return StatusCodes.FORBIDDEN;
```

Comment:
Forbidden is for user authorization problems. Conflict (409) probably makes the most sense?

Reply:
Yeah. That makes sense. Changed.